src/pages/Payment/Payment.jsx

```diff
@@ -1,8 +1,37 @@
-import React from 'react';
-import './Payment.scss';
+import React, { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
+import './Payment.scss';
 
 const Payment = () => {
+  const [productList, setProductList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/data.json', {
+      // fetch('http://10.58.52.239:8000/cart', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+    })
+      .then(res => res.json())
+      .then(data => {
+        setProductList(data.data);
+      });
+  }, []);
+
+  const totalPrice = () => {
+    const sumPrice = productList
+      .map(list => list.total_price)
+      .reduce((acc, cur) => acc + cur, 0);
+
+    const sumDeliveryCharge = productList.reduce((acc, cur) => {
+      if (cur.total_price < 50000) {
+        return acc + 3000;
+      } else return acc;
+    }, 0);
+    return sumPrice + sumDeliveryCharge;
+  };
```

Comment:
단순히 변수 두 개를 더해서 return해주는 함수라면, 굳이 함수로 만들 필요 없이 변수 자체로 선언하고 사용하시면 될 것 같습니다.
```suggestion
  const sumPrice = productList
    .map(list => list.total_price)
    .reduce((acc, cur) => acc + cur, 0);

  const sumDeliveryCharge = productList.reduce((acc, cur) => {
    if (cur.total_price < 50000) {
      return acc + 3000;
    } else return acc;
  }, 0);
  
  const totalPrice = sumPrice + sumDeliveryCharge;
```